src/pages/networks/NetworkList.tsx

```diff
@@ -80,8 +80,9 @@ const NetworkList: FC = () => {
     isLoading: isClusterNetworksLoading,
   } = useQuery({
     queryKey: [queryKeys.networks, project, queryKeys.cluster],
-    queryFn: () => fetchNetworksFromClusterMembers(project, clusterMembers),
-    enabled: clusterMembers.length > 0,
+    queryFn: () =>
+      fetchNetworksFromClusterMembers(project, clusterMembers, isFineGrained),
+    enabled: clusterMembers.length > 0 && isFineGrained !== null,
```

Comment:
I think this should be part of the hook as well. Then we can contain the `isFineGrained` call in it.